po/hi.po

```diff
@@ -222,52 +224,62 @@ msgstr "विवरण (खाली)"
 msgid "Name (Exists)"
 msgstr "नाम (पहले से मौजूद है)"
 
-#: src/ui/views/mainwindow.cpp:27 src/ui/views/mainwindow.cpp:58
-#: src/ui/views/shortcutsdialog.cpp:96
-msgid "New Account"
-msgstr "नया खाता"
+#: src/ui/views/mainwindow.cpp:28
+msgid "Recents"
+msgstr "हाल ही के"
 
-#: src/ui/views/mainwindow.cpp:28 src/ui/views/mainwindow.cpp:66
-#: src/ui/views/mainwindow.cpp:157 src/ui/views/mainwindow.cpp:185
-#: src/ui/views/shortcutsdialog.cpp:97
-msgid "Open Account"
-msgstr "खाता खोलें"
+#: src/ui/views/mainwindow.cpp:41
+msgid "New"
+msgstr "नया"
 
-#: src/ui/views/mainwindow.cpp:29 src/ui/views/shortcutsdialog.cpp:98
-msgid "Close Account"
-msgstr "खाता बंद करें"
+#: src/ui/views/mainwindow.cpp:44
+msgid "New Account (Ctrl+N)"
+msgstr "नया खाता (Ctrl+G)"
```

Comment:
@heymisphere keyboard shortcut is wrong.

Reply:
ah sorry, would correct it now, i know what caused it. The string was previously 'New' which used to refer to group, but due to fuzzy function, 'new' is part of 'new account' so it just kept it like that, and though i changed it to account, i forgot to change the keyboard shorcut